### ISSUE TITLE
machinectl clone error should be more descriptive

### ISSUE THREAD

**Issue by xor-gate** [systemd/systemd#2060]
While i'm very new to systemd machinectl + systemd-nspawn (migrating from lxc) I got a very weird error which took me some time:
- ArchLinux host (systemd 228)
- /var/lib/machine btrfs filesystem

I then bootstrapped a fresh arch container directly into the /var/lib/machine.

```
# machinectl list-images
NAME        TYPE      RO  USAGE  CREATED                     MODIFIED
arch01      directory no  n/a    n/a                         n/a
```

Then I tried to clone:

```
# machinectl clone arch01 arch01-copy
Could not clone image: Operation not supported
```

When I tarred the `arch01` and then `machinectl import-tar ./arch01.tar.bz2 test-arch01` it works fine:

```
# machinectl list-images
NAME        TYPE      RO  USAGE  CREATED                     MODIFIED
arch01      directory no  n/a    n/a                         n/a
test-arch01 subvolume no  626.1M Sun 2015-11-29 13:50:16 CET n/a
```

```
# machinectl clone test-arch01 clone-arch01
# machinectl list-images
NAME         TYPE      RO  USAGE  CREATED                     MODIFIED
arch01       directory no  n/a    n/a                         n/a
clone-arch01 subvolume no  630.8M Sun 2015-11-29 13:51:46 CET n/a
test-arch01  subvolume no  630.8M Sun 2015-11-29 13:50:16 CET n/a
```

My suggestion would be a extra check if the clone is on a btrfs filesystem where the containers is not on a subvolume. And error something like: `Could not clone image: image not loaded in a btrfs subvolume`.

See also (where I found my answer): https://propellor.branchable.com/forum/Systemd.container_produces_non-standard_systemd_container/

**Comment by poettering** [comment]
Fix waiting in #3117.

**Comment by xor-gate** [comment]
Thanks @poettering for fixing 👍
